web/src/hooks/useToken.ts

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+const UseToken: any = () => {
```

Comment:
lower case. Im not sure if we need this hook. you can remove it

you can create util functions - getLocalStorage, saveLocalStorage